data/data/aws/master/main.tf

```diff
@@ -33,41 +29,6 @@ EOF
   ), var.tags)}"
 }
 
-resource "aws_iam_role_policy" "master_policy" {
```

Comment:
We need the iam policy for kubelet to running. also the kube-controller-manager would need access to AWS api too.

Reply:
I removed the change, something like this can happen in another PR if wanted.